tests/Support/Traits/CssDataProviders.php

```diff
@@ -27,7 +27,7 @@ public function provideEquivalentCss(): array
      */
     public function provideEquivalentCompleteCss(): array
     {
-        $equivalentCssWithAtMediaRuleSelectorListAndPropertyDeclaration = [
+        $datasetsWithAtMediaRuleSelectorListAndPropertyDeclaration = static::crossDatasetWithItself([
```

Comment:
Could we also use `self::` instead of `static::` in all changed places?

Reply:
We can.  Have changed it.